app-web/__tests__/pages/Index.test.js

```diff
@@ -123,6 +123,7 @@ describe('Home Page', () => {
     queryString.parse.mockReturnValue({ q: 'foo' });
     // rerender stubbing no results for resources
     useSearch.mockReturnValue([]);
+    removeUnwantedResults.mockReturnValue([]);
```

Comment:
For some mystical reason, if removeUnwantedResults is imported from helper into page/index(or anywhere else),
it will return [] as undefined in the initial render. so I just mock the return value here to pass the test as in this case, useSearch is already empty array.